tmp/lanelet2_extension/lib/visualization.cpp

```diff
@@ -552,7 +554,7 @@ visualization_msgs::msg::MarkerArray visualization::generateTrafficLightIdMaker(
         marker.header.frame_id = "map";
         marker.header.stamp = rclcpp::Time();
         marker.ns = "traffic_light_id";
-        marker.id = static_cast<int32_t>(ls.id());
```

Comment:
Same here, all the linestrings in Lanelet2 map should be unique and shouldn't have duplication.